Langs/1065.lang

```diff
@@ -0,0 +1,59 @@
+[MetaData]
+Language=Persian - Iran
+Translator=Seyed Omid Mazlomy
+Version=2.3.0.0
+[Strings]
+Donate=حمایت مالی
+Check for Updates=بررسی به روز رسانی
+Your Windows 11 Compatibility Results are Below=نتایج امکان سنجی نصب ویندوز 11
+Now Reach WhyNotWin11 via https://www.whynotwin11.org/=حالا از طریق لینک http://www.whynotwin11.org به ما دسترسی داشته باشید
+Results Based on Currently Known Requirements!=نتایج بر اساس حداقل مشخصات فعلی
+Translation by=ترجمه توسط
+Architecture (CPU + OS)=معماری سیستم ( ویندوز + سی پی یو)
+Boot Method=روش بوت
+CPU Compatibility=سازگاری سی پی یو
+CPU Core Count=تعداد هسته سی پی یو
+CPU Frequency=فرکانس سی پی یو
+DirectX + WDDM2= دایرکت ایکس + WDDM2
+Disk Partition Type=نوع پارتیشن دیسک
+RAM Installed=رم نصب شده
+Secure Boot=بوت امن
+Storage Available=حافظه در دسترس
+TPM Version=ورژن TMP
+Checking...=در حال بررسی ...
+64 Bit CPU=سی پی یو 64 بیتی
+64 Bit OS=ویندوز 64 بیتی
+32 Bit CPU=سی پی یو 32 بیتی
+32 Bit OS=ویندوز 32 بیتی
+Unable to Determine=امکان تشخیص نیست
+Unable to Check List=امکان چک کردن لیست نیست
+Error Accessing List=مشکل دسترسی به لیست
+Not Currently Listed as Compatible=در حال حاضر در لیست سازگاری نیست
+Listed as Compatible=سازگار می باشد
+Cores=هسته ها
+Threads=رشته
+GPT Detected=GPT یافت شد
+GPT Not Detected=GPT یافت نشد
+Enabled=فعال
+Supported=پشتیبانی میشود
+Not Supported=پشتیبانی نمیشود
+Disabled / Not Detected=غیر فعال / شناسایی نشد
+Drive(s) Meet Requirements=درایو ها حداقل را پشتیبانی میکند
+TPM Missing / Disabled=TPM یافت نشد / غیر فعال است
+Detected=شناسایی شد
+DirectX 12 and WDDM 3=دایرکت ایکس 12 و WDDM نسخه 3
+DirectX 12 and WDDM 2=دایرکت ایکس 12 و WDDM نسخه 2
+No DirectX 12, but WDDM2=دایرکت ایکس 12 نیست اما WDDM2 است
+DirectX 12, but no WDDM2=دایرکت ایکس 12 است اما WDDM2 نیست
+No DirectX 12 or WDDM2=دایرکت ایکس 12 و WDDM2 نیست
+Test Build?=تست نسخه؟
+You're running a newer build than publically Available!=شما نسخه جدیدتری از عموم را استفاده میکنید
+Up to Date=به روز
+You're running the latest build!=آخرین ورژن را استفاده میکنید
+Unable to Check for Updates=امکان چک کردن آپدیت نیست
+Unable to load release data.=اطلاعات ریلیز را لود نکرد
+Invalid Data Received!=دیتا با خطا دریافت شد
+Invalid Release Tags Received!=تگ های ریلیز با خطا دریافت شد
```

Comment:
Hello, I quickly check it with google translate and it seems like you did phonetic "release"? Could you explain?
![image](https://user-images.githubusercontent.com/45581170/123800901-0a331980-d8ea-11eb-8f92-7a1f0197bd28.png)

Reply:
hi, this is an English word which there isn't any good translation in Persian, so we translate it phonetically.

Reply:
absolutely

Reply:
You can check Persian translation in my fork, I have MA in English studies and worked as TEFL teacher for 5 years. Translations in my fork are free to use, and those are actual words we use not robotic translations. Thanks